### PR TITLE
update company field

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe, look ok?

### CODE REVIEW

**Comment by alexkwolfe** [comment]
You'll need to make `company` a deprecated field.

**Comment by alexkwolfe** [comment]
That will also mean changes to the integrations that support it.

**Comment by ros3bud** [comment]
@alexkwolfe, company is now deprecated. I have not yet updated the version though, as we talked about.

File: fields/company.json [hunk 1]
```diff
@@ -8,4 +8,5 @@
 {"id":"company.monthly_sales",    	"name":"Company Monthly Sales",     	"type":"range",   		"description":"Company Monthly Sales"}
 {"id":"company.type",             	"name":"Company Type",              	"type":"string",  		"description":"Business to Consumer, Business to Business, Business to Government"}
 {"id":"company.accounts_receivable","name":"Company Accounts Receivable",   "type":"range",   		"description":"Accounts Receivable amount"}
-{"id":"company.monthly_charges",	"name":"Company Monthly Charges",   	"type":"range",   		"description":"Monthly charges via credit cards"}
\ No newline at end of file
+{"id":"company.monthly_charges",	"name":"Company Monthly Charges",   	"type":"range",   		"description":"Monthly charges via credit cards"}
+{ "deprecated": true, "see": "company", "id": "company.name", "name": "company", "type": "string",       "description":"Name of company/employer"}
```
Comment (cgrayson) [inline fields/company.json]: The `id` and `name` attributes are reversed. `id` is the old name.

File: fields/company.json [hunk 2]
```diff
@@ -8,4 +8,5 @@
 {"id":"company.monthly_sales",    	"name":"Company Monthly Sales",     	"type":"range",   		"description":"Company Monthly Sales"}
 {"id":"company.type",             	"name":"Company Type",              	"type":"string",  		"description":"Business to Consumer, Business to Business, Business to Government"}
 {"id":"company.accounts_receivable","name":"Company Accounts Receivable",   "type":"range",   		"description":"Accounts Receivable amount"}
-{"id":"company.monthly_charges",	"name":"Company Monthly Charges",   	"type":"range",   		"description":"Monthly charges via credit cards"}
\ No newline at end of file
+{"id":"company.monthly_charges",	"name":"Company Monthly Charges",   	"type":"range",   		"description":"Monthly charges via credit cards"}
+{ "deprecated": true, "see": "company", "id": "company", "name": "company.name", "type": "string",       "description":"Name of company/employer"}
```
Comment (cgrayson) [inline fields/company.json]: Shoot. Sorry about that, I misspoke. `name` stays as the textual description. It's `see` that should be set to the new name. :disappointed: The [mortgage fields file](https://github.com/activeprospect/leadconduit-fields/blob/master/fields/mortgage.json) has examples.

**Comment by ros3bud** [comment]
@cgrayson or @alexkwolfe is this ok to merge?

File: fields/company.json (hunk 2 above)
Comment (alexkwolfe) [inline fields/company.json]: You want 
```json
"see": "company.name", "id": "company", "name": "Company Name"
```

**Comment by cgrayson** [comment]
LGTM :-)

**Comment by alexkwolfe** [comment]
Me too. :balloon:

**Comment by ros3bud** [comment]
Thanks!

**Pull request merged**